### DISCUSSION TITLE
Modal component with x-modelable and text input causes flashing and other strange behavior

### DISCUSSION THREAD

**Discussion by betterrx-developers** [alpinejs/alpine#2686] [7. Bugs (archived)]
We have a modal component using the new x-modelable tag.  We noticed when a user types into a text input in the modal window, the overlay starts flashing and the modal window starts vibrating. If you delete all of the text out of the text input, the modal window closes.

You can see this behavior with the following [code pen](https://codepen.io/cby016/pen/PoOQEjE)

**Comment by SimoTod** [comment]
Another user reported a similar issue. It's because x-model reacts to any input event bubbling up the dom from the internal text field. Until Caleb clarify how it should be used, you can put `@input.stop` on the text field to stop the event.

**Reply by betterrx-developers** [reply]
@SimoTod That seems to fix the issue. Thanks for the quick response.
